core/session.py

```diff
@@ -69,18 +65,17 @@ def load(self, req: BrowseHandler):
             If the client supplied a valid Cookie, the session is read from the datastore, otherwise a new,
             empty session will be initialized.
         """
-        if self.cookieName in req.request.cookies:
-            cookie = str(req.request.cookies[self.cookieName])
+        if self.cookie_name in req.request.cookies:
+            cookie = str(req.request.cookies[self.cookie_name])
```

Comment:
```suggestion
        if self.cookie_key := str(req.request.cookies.get(self.cookie_name)):
```
I think we can simplify this.